### ISSUE TITLE
[feature] Enable lint options

### ISSUE THREAD

**Issue by genebean** [declarativesystems/onceover-codequality#5]
Is there any way to override these?

```ruby
    LINT_OPTIONS = [
        "--relative",
        "--fail-on-warnings",
        "--no-double_quoted_strings-check",
        "--no-80chars-check",
        "--no-variable_scope-check",
        "--no-quoted_booleans-check",
      ]
```

If not, can you enable overriding them?

**Comment by GeoffWilliams** [comment]
Plan would be to allow `.puppet-lint.rc` to override these default settings if existing

**Comment by GeoffWilliams** [comment]
released in 0.4.0
